v2/pkg/protocols/http/request_annotations.go

```diff
@@ -62,9 +61,9 @@ func (r *Request) parseAnnotations(rawRequest string, request *http.Request) (*h
 		// handle scheme
 		switch {
 		case stringsutil.HasPrefixI(value, "http://"):
```

Comment:
The idea of defining these as reusable constants in utils library didn't seem bad:
```
http://
https://
http
https
```
What do you think?